modules/search/pages/field-data-types-reference.adoc

```diff
@@ -0,0 +1,36 @@
+= Field Data Types 
+:page-topic-type: reference 
+:description: You can assign a data type to a field to tell the Search Service how to analyze its data. 
+
+[abstract]
+{description}
+
+When you xref:create-child-field.adoc[create a child field] on a type mapping or xref:create-quick-index.adoc[], you need to set a field's data type. 
+
+The following field data types are available: 
+
+[cols="1,2"]
+|====
+|Field Data Type |Description 
+
+|text |The field contains a string. The string can contain numbers and special characters.
+
+|number |The field contains a number. It doesn't contain any alphabetic characters. 
+
+|datetime |The field contains a date/time value that matches the format of a xref:[Date/Time Parser] in the index. 
+
+|boolean |The field contains a true or false value. 
+
+|disabled | This field data type is deprecated. It's included for compatibility only. 
+
+|geopoint a|
+
+The field contains geopoint (latitude and longitude) data, represented as either: 
+
+* A string, as two numeric values separated by a comma. 
+* A string, as a geohash point. 
+* An array, as two floating point integers. 
+* A JSON object, as the properties `"lon"`/`"lng"` and `"lat"`.
```

Comment:
"*as* the properties" ?

you don't need the string quotes for the properties do you?